about.md

```diff
@@ -1,36 +1,35 @@
 # Noi App
 
-This project aims to create a contact tracing mobile based on the [DP-3T](https://github.com/DP-3T) standard, specifically aimed at the Italian population.
+Lo scopo del progetto è creare un app mobile di tracciamento dei contatti basata sullo standard [DP-3T](https://github.com/DP-3T), specificamente per la popolazione Italiana
 
-This standard is designed to be privacy respectful and does not share information, except anonymized data for voluntary disclosure by people who find themself infected. Those data can then be checked by the app to verify if you were in contact with someone infected.
+Lo standard è disegnato per rispettare la privacy per non condividere informazioni, se non dati anonimizzati per divulgazione volontaria di persone che hanno scoperto di essere infette. Questi dati possono essere controllati dall'app per verificare se sei stato in contatto con qualche persona infetta.
```

Comment:
Lo standard è disegnato per rispettare la privacy e non condivide informazioni, tranne dati anonimizzati per divulgazione volontaria di persone che hanno scoperto di essere infette. Questi dati possono essere controllati dall'app per verificare se sei stato in contatto con qualche persona infetta.